spec/support/shared_examples/sign_in.rb

```diff
@@ -280,6 +280,11 @@ def no_authn_context_sign_in_with_piv_cac_goes_to_sp(sp)
 
   fill_in_password_and_submit(user.password)
 
+  # needed because the SP default attribute bundle includes the zip_code
```

Comment:
This test was actually broken on `main` - we should be requesting an extra consent on the second request since the `address` scope wasn't included in the original enrollment request but we're not since our `SamlRequestPresenter` just defaults to `email` unless IAL2 is explicitly requested (not correct, I don't think).